### ISSUE TITLE
Update to support a local file server instead of ThePosterDB

### ISSUE THREAD

**Issue by jeffreyswiggins** [bbrown430/plex-poster-set-helper#32]
So could this be modified to support a local http/s type file server that could be running on my network storing copies of I have stored on the artwork I have downloaded from the sites as a backups.  I have started using these scripts to write them to my Plex Metadate data and remove them from its direct file store and just storing them in another file store location as backup in case another "PosterDB Outage" happens for months so I have access to the original data.  When I try to point the script to use somethings like the DUFS docker container to a local http:// address (I do not want it externally published ) I get an error of:

plexapi.exceptions.BadRequest: (406) not_acceptable; http://xxxxx.xxxxxx.local:32400/library/metadata/1142869/posters?url=http%3A%2F%2Flocal.site.address%3A4688%2Fsata1%2F57Seconds.2023%2F57Seconds.jpg <html><head><title>Not Acceptable</title></head><body><h1>406 Not Acceptable</h1></body></html>

I would love to use this to simply write the local store I have to my Plex Metadata and going forward just use another trick I have for downloading the artwork from those sites in more of an automated way and to this file store and use this to just "set them" into plex metadata.

Just thought i gets this nice script working again.  I know it works MediUX for now but who knows for how long they allow that

**Comment by bbrown430** [comment]
@jeffreyswiggins I am not sure if this is the same as what you are suggesting but in the latest commit I have added support for local HTML files. For instance, you can now pass the path of an HTML file, e.g. posters/marvel.html and it will function as expected.

**Comment by jeffreyswiggins** [comment]
Sorry it took me a while to respond.  Is there a document on how to use this?  I looked at the commit, but need a little bit of documentation on the html file.  I have an local archive of PNG and JPGs after the The PosterDB multi-month hiatus that are organized and I could create a yaml (or I guess HTML file of local file links) if that is the intention. Just not sure how this is supposed to be utilized.

Thanks

**Comment by giantsfan006** [comment]
> Just not sure how this is supposed to be utilized.

Ever forget figure it out? I'm looking to do something similar.

**Comment by jeffreyswiggins** [comment]
> > Just not sure how this is supposed to be utilized.
> 
> Ever forget figure it out? I'm looking to do something similar.

no... i pulled the latest version and tried a few things but not what i was expecting with the way of results, and the description of local HTML files is.  I am assuming it is for me to "build" a HTML file that lists all the files and directory path on my file server like a "file list" and it will use it to pull it.  I have not had luck with the file.

**Comment by jeffreyswiggins** [comment]
> [@jeffreyswiggins](https://github.com/jeffreyswiggins) I am not sure if this is the same as what you are suggesting but in the latest commit I have added support for local HTML files. For instance, you can now pass the path of an HTML file, e.g. posters/marvel.html and it will function as expected.

okay i wanted to come back to this.  I think what you gave was the ability to use a source HTML file as the "config" for it to use to map the posters to the files in PLex, but what I was asking about is I have all my poster files stored locally and would like to point this to a local UNC path for the posters to be retrieved and added to plex from, not using MediUX or the PosterDB as the poster file source.  Hope this makes better sense and you can help.
